tests/layer_tests/tensorflow_tests/test_tf_Reciprocal.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+
+
+class TestReciprocal(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        assert 'x' in inputs_info
+        x_shape = inputs_info['x']
+        inputs_data = {}
+        inputs_data['x'] = np.random.randint(1, 30, x_shape).astype(np.float32)
+
+        return inputs_data
+
+    def create_reciprocal_net(self, x_shape, x_type):
+        tf.compat.v1.reset_default_graph()
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            x = tf.compat.v1.placeholder(tf.float32, x_shape, 'x')
+            tf.raw_ops.Reciprocal(x=x)
```

Comment:
Could you check, this test looks like unary test like a listed in test_tf_UnaryOps.py. I understand when we test different dtypes, but in case of single float32 maybe better to extend UnaryOps.py

Reply:
Let me fix in the different PR. The idea originally was to test `int32` type as well but TF does not support it:)